webapp/javascript/components/SideTimelineComparator/index.tsx

```diff
@@ -0,0 +1,224 @@
+import React, { useEffect, useRef, useState } from 'react';
+import classNames from 'classnames/bind';
+import Button from '@webapp/ui/Button';
+import { Popover, PopoverBody } from '@webapp/ui/Popover';
+import { Portal } from '@webapp/ui/Portal';
+import { faChevronDown } from '@fortawesome/free-solid-svg-icons/faChevronDown';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  markingsFromSelection,
+  Selection,
+} from '@webapp/components/TimelineChart/markings';
+import styles from './styles.module.scss';
+
+interface SideTimelineComparatorProps {
+  onCompare: (params: {
+    from: string;
+    until: string;
+    leftFrom: string;
+    leftTo: string;
+    rightFrom: string;
+    rightTo: string;
+  }) => void;
+  selection: {
+    left: Selection;
+    right: Selection;
+  };
+}
+
+const cx = classNames.bind(styles);
+
+const buttons = [
+  [
+    {
+      label: '1 hour prior',
+      ms: 3600 * 1000,
+    },
+    {
+      label: '12 hour prior',
+      ms: 43200 * 1000,
+    },
+    {
+      label: '24 hour prior',
+      ms: 86400 * 1000,
+    },
+  ],
+  [
+    {
+      label: '1 week prior',
+      ms: 604800 * 1000,
+    },
+    {
+      label: '2 weeks prior',
+      ms: 1209600 * 1000,
+    },
+    {
+      label: '30 days prior',
+      ms: 2592000 * 1000,
+    },
+  ],
+];
```

Comment:
Just thinking out loud, maybe it would be more useful if times are something shorter like:
| 1 hour | 12 hours |
| 2 hours | 24 hours |
| 6 hours | 1 week |


@kolesnikovae @eh-am @korniltsev  What do you guys think?